scripts/scripts.js

```diff
@@ -35,7 +35,7 @@ export function sampleRUM(checkpoint, data = {}) {
       const sendPing = () => {
         // eslint-disable-next-line object-curly-newline, max-len, no-use-before-define
         const body = JSON.stringify({ weight, id, referer: window.location.href, generation: RUM_GENERATION, checkpoint, ...data });
-        const url = `https://rum.hlx3.page/.rum/${weight}`;
+        const url = `https://rum.hlx.page/.rum/${weight}`;
```

Comment:
Not sure about that. Let me test.

Reply:
Now it works.

Reply:
@stefan-guggisberg fyi, added the domain in https://manage.fastly.com/compute/services/5Qir8H8bLeaRuldIp9TWq4/versions/39/domains